README.md

```diff
@@ -1,56 +1,50 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# MATH MAGICIANS
 
+> Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a random math-related quote.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
+- React
+- Tailwind CSS
 
-[Live Demo Link](https://livedemo.com)
+## Live Demo
 
+[Live Demo Link](https://mwafrika.github.io/math_magicians/)
 
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+`Node must be installed in your local for your project to run`
+
 ### Setup
 
+`git clone https://github.com/mwafrika/math_magicians.git && cd cloned_repository`
+
 ### Install
 
+`npm install`
+
 ### Usage
 
 ### Run tests
 
 ### Deployment
```

Comment:
- [ ] In your README, kindly add instructions on how to run the app locally i.e. `npm start`.

- [ ] Remove unnecessary sections from your README that have no descriptive body yet.